### ISSUE TITLE
GCE terraform refresh google_compute_instance does not refresh machine_type

### ISSUE THREAD

**Issue by allandrick** [hashicorp/terraform#11552]
### Affected Resource(s)
Please list the resources as a list, for example:
- google_compute_instance

### Terraform Configuration Files
```hcl
resource "google_compute_instance" "testInstance" {
  name = "testInstance"
  description = "testInstance"
  machine_type = "custom-1-6144"
  zone = "europe-west1-b"

  lifecycle {
    ignore_changes = [
      "create_timeout"
    ]
  }

  disk {
    disk = "disk-1"
    device_name = "disk1"
    auto_delete = "false"
  }

  network_interface {
    subnetwork = "somewhere-europe-west"
  }

  can_ip_forward = "false"
}
```

### Debug Output

Please contact me privately for Gist link, but it is easily repeatable, see below.

### Panic Output:

none

### Expected Behavior

https://www.terraform.io/docs/commands/refresh.html

reconcile the state Terraform knows about (via its state file) with the real-world infrastructure. This can be used to detect any drift from the last-known state, and to update the state file.

### Actual Behavior

The state file is not updated from the real-world infrastructure.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. deploy a google_compute_instance with terraform
2. edit the machine-type using GCP's console's sizing recommendations tool (https://cloud.google.com/compute/docs/instances/viewing-sizing-recommendations-for-instances)
3. `terraform refresh` - does not update state from GCP

### Important Factoids

https://cloud.google.com/compute/docs/instances/viewing-sizing-recommendations-for-instances

is a new feature (currently listed as Beta, but widely adopted) - so there may have been an assumption in the Terraform code for google_compute_instance to not need to refresh normally un-editable resource attributes via the console.

### References

None found.

**Comment by mbfrahry** [comment]
Hey @allandrick! Thanks for finding and posting this issue. We just merged a fix for this, PR #11645

**Comment by allandrick** [comment]
Hi @mbfrahry,

Do you happen to know which release will include this PR?  I checked the changelog on various tags but couldn't see a reference to it.

Thanks!
